src/index.js

```diff
@@ -165,19 +170,21 @@ OAuth.prototype.refresh = function (url, callback) {
  * @param  {String}   clientId The client id
  * @param  {Function} callback First arg is error (if any), second is OAuth object on success.
  */
-OAuth.authorizeImplicitly = function(url, clientId, callback) {
-	var self = this,
-		win,
+OAuth.authorizeImplicitly = function(url, clientId, scopes, callback) {
+    if (typeof scopes === 'function') {
+        callback = scopes;
+        scopes = ''
+    }
+	var win,
 		webview,
-		retryCount = 0,
 		state = generateGUID();
 	win = Ti.UI.createWindow(AUTH_WINDOW_OPTIONS);
 	webview = Ti.UI.createWebView({
 		width : Ti.UI.FILL,
 		height : Ti.UI.FILL,
-		ignoreSslError: this.ignoreSslError,
+		ignoreSslError: this.ignoreSslError ? true : false,
```

Comment:
Shouldn't need to do the truth check with true/false ternary. Just assign the value